### PR TITLE
Initial setup instructions

### PR DESCRIPTION
**Pull request by dbrand666**
Added instructions for initial setup without the Reolink app or Internet access.

## Proposed change

I had a brand new Reolink doorbell and noticed that the setup instructions seem to assume that the device is already online. I wanted to avoid using the Reolink app and didn't want to attach it to my LAN. A bit of searching turned up a 3 year old Reddit post that never got any traction. It's a very quick and simple way to connect a new device to a Wi-Fi network.

I think it's worth documenting it in here.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/reolink.markdown [hunk 1]
```diff
@@ -204,6 +204,18 @@ However, these cameras can work with this integration through an NVR in which th
 - E1 Pro
 - E1
 
+## Initial Setup
+
+You'll need to configure your new camera/doorbell for your network. If you're using a wired LAN this is simple enough. Simply plug it in and let it use DHCP to set itself up. Once it's recognized by your network you can access its configuration with a web browser.
```
Comment (c0ffeeca7) [inline source/_integrations/reolink.markdown]:
```suggestion
You'll need to configure your new camera/doorbell for your network. If you're using a wired LAN, this is simple enough. Simply plug it in and let it use DHCP to set itself up. Once it's recognized by your network, you can access its configuration with a web browser.
```

File: source/_integrations/reolink.markdown [hunk 2]
```diff
@@ -204,6 +204,18 @@ However, these cameras can work with this integration through an NVR in which th
...
+
+If you prefer to use Wi-Fi, especially if you want to retrict it to a local IoT network, [here's a quick way to get it connected](https://www.reddit.com/r/reolink/comments/hjn5be/how_to_connect_wifi_without_an_app/). Simply create a QR code with the following XML string:
```
Comment (c0ffeeca7) [inline source/_integrations/reolink.markdown]:
```suggestion
If you prefer to use Wi-Fi, especially if you want to restrict it to a local IoT network, [here's a quick way to get it connected](https://www.reddit.com/r/reolink/comments/hjn5be/how_to_connect_wifi_without_an_app/). Simply create a QR code with the following XML string:
```

File: source/_integrations/reolink.markdown [hunk 3]
```diff
@@ -204,6 +204,18 @@ However, these cameras can work with this integration through an NVR in which th
...
+
+    <QR><S>ssid</S><P>password</P><C>last4</C></QR>
+
+Use the `ssid` and `password` of your IoT network. The `last4` is the last 4 digits of the QR code used for the Reolink setup - it's printed under the QR code or simply scan the QR code and grab the last 4 digites.
```
Comment (c0ffeeca7) [inline source/_integrations/reolink.markdown]:
```suggestion
Use the `ssid` and `password` of your IoT network. The `last4` is the last 4 digits of the QR code used for the Reolink setup. It's printed under the QR code. You can also scan the QR code and grab the last 4 digits.
```

**Review by c0ffeeca7 — CHANGES_REQUESTED** [review]

**Review by c0ffeeca7 — APPROVED** [review]
Nice! Thank you, @dbrand666 :+1:

File: source/_integrations/reolink.markdown [hunk 4]
```diff
@@ -204,6 +204,18 @@ However, these cameras can work with this integration through an NVR in which th
 - E1 Pro
 - E1
 
+## Initial Setup
+
+You'll need to configure your new camera/doorbell for your network. If you're using a wired LAN, this is simple enough. Simply plug it in and let it use DHCP to set itself up. Once it's recognized by your network, you can access its configuration with a web browser.
+
+If you prefer to use Wi-Fi, especially if you want to restrict it to a local IoT network, [here's a quick way to get it connected](https://www.reddit.com/r/reolink/comments/hjn5be/how_to_connect_wifi_without_an_app/). Simply create a QR code with the following XML string:
```
Comment (frenck) [inline source/_integrations/reolink.markdown:211]: We should avoid the term `Simply` in general. What is simple for you, doesn't have to be simple for someone else and thus might come across as condescending.

I'm not sure if the reddit link provides any additional value?

File: source/_integrations/reolink.markdown [hunk 5]
```diff
@@ -204,6 +204,18 @@ However, these cameras can work with this integration through an NVR in which th
 - E1 Pro
 - E1
 
+## Initial Setup
+
```
Comment (starkillerOG) [inline source/_integrations/reolink.markdown:208]: I would first recommend users to just use the Reolink app/windows/mac client and follow the instructions there. That is by far the easiest and what most users are going to do. Once the setup is complete and the username and password of the camera is set in the Reolink client, use those same credentials inside Home Assistant.

Then as a second option for camera's that have a LAN port (most WiFi camera's also have a LAN port), connect the camera to your network using a LAN cable and browse to the IP adress of the camera. Follow the instructions to create the user credentials (needed in HomeAssistant). Then in the settings of the camera, configure the wifi SSID and password, once configured disconnect the LAN cable and the camera will automatically switch to the wifi connection if the wifi credentials are correct.

Then we can explain how to do it if someone insists on not using the Reolink app/windows/mac client and has a camera withouth a LAN port as you discribed.

File: source/_integrations/reolink.markdown (hunk 4 above)
Comment (starkillerOG) [reply source/_integrations/reolink.markdown:211]: I agree.

There is no additional info in the reddit post, so I think we can remove it

Comment (starkillerOG) [reply source/_integrations/reolink.markdown:211]: So actually I don't even know how to make a QR code.
Is this something build in to many phones or do you need a app/online tool to create QR codes?

File: source/_integrations/reolink.markdown [hunk 6]
```diff
@@ -204,6 +204,18 @@ However, these cameras can work with this integration through an NVR in which th
...
+You'll need to configure your new camera/doorbell for your network. If you're using a wired LAN, this is simple enough. Simply plug it in and let it use DHCP to set itself up. Once it's recognized by your network, you can access its configuration with a web browser.
+
+If you prefer to use Wi-Fi, especially if you want to restrict it to a local IoT network, [here's a quick way to get it connected](https://www.reddit.com/r/reolink/comments/hjn5be/how_to_connect_wifi_without_an_app/). Simply create a QR code with the following XML string:
+
+    <QR><S>ssid</S><P>password</P><C>last4</C></QR>
+
+Use the `ssid` and `password` of your IoT network. The `last4` is the last 4 digits of the QR code used for the Reolink setup. It's printed under the QR code. You can also scan the QR code and grab the last 4 digits.
```
Comment (starkillerOG) [inline source/_integrations/reolink.markdown:215]: > the QR code used for the Reolink setup

Where can a user find this QR code?

**Review by starkillerOG — COMMENTED** [review]
I would like to make some changes to this documentation, (see my comments).
Therefore I will submit a seprate PR.

Could you explain how to make the QR code and where to find the QR code for the "last 4 digits"?

**Comment by starkillerOG** [comment]
I made some changes to clearify it a bit in this PR: https://github.com/home-assistant/home-assistant.io/pull/27722

**Pull request merged**